scheduler/src/cook/mesos/api.clj

```diff
@@ -551,6 +557,108 @@
                           cheshire/generate-string)))
       ring.middleware.json/wrap-json-params))
 
+
+
+(defn retries
```

Comment:
This isn't necessary as this info can be gotten by querying the job endpoint. Up to you if you want to add it in the liberator form.